operator/cmd/mesh/manifest-apply.go

```diff
@@ -160,7 +169,24 @@ func ApplyManifests(setOverlay []string, inFilenames []string, force bool, dryRu
 		l.logAndPrint("\n\n✘ Errors were logged during apply operation. Please check component installation logs above.\n")
 		return fmt.Errorf("errors were logged during apply operation")
 	}
-
 	l.logAndPrint("\n\n✔ Installation complete\n")
+
+	// TODO: modify this to the passed-in revision.
```

Comment:
Why? its on line 148

Reply:
Is the plan to do -f <my file> --set revision=foo? i saw a --revision flag being floated around.

Reply:
Yes its `--set revision`, its part of the IOP